Source/Eto/Drawing/PaddingConverter.cs

```diff
@@ -12,6 +12,11 @@ namespace Eto.Drawing
 	public class PaddingConverter : TypeConverter
 	{
 		/// <summary>
+		/// The character to split up the string which will be converted
+		/// </summary>
+		private readonly string[] StringSplitter = new string[1] { "," };
```

Comment:
These could be static in each of the classes, and you don't need to use the 'private' modifier as it is redundant.